### ISSUE TITLE
Error if comment put in a nested declaration (contents of namespaced properties must result in style declarations only)

### ISSUE THREAD

**Issue by mzgol** [sass/libsass#890]
With grunt-sass `0.18.0` (using node-sass `2.0.1` which relies on libsass `3.1.0`) the following SCSS:
```scss
.clz {
    border: {
        right: 10px solid /*color is set in the theme*/;
    }
}
```
results in an error:
```
contents of namespaced properties must result in style declarations only
Backtrace:
	styles.scss:3
  Line 3  Column 27  styles.scss
```
It works fine with Sass 3.4.12. Removing the comment makes the error go away.

**Comment by xzyfer** [comment]
I can confirm this error is present in Libsass master.

**Comment by xzyfer** [comment]
Spec added https://github.com/sass/sass-spec/pull/265/files

**Comment by xzyfer** [comment]
This is fixed and will be in 3.2.
